### ISSUE TITLE
Better document our web services use

### ISSUE THREAD

**Issue by Frijol** [tessel/project#345]
List the services we're using in https://github.com/tessel/project/blob/master/WWW.md and how they map on (e.g. Slackin)

**Comment by tcr** [comment]
All these are hosted on Dokku:

* docs.tessel.io and doc.tessel.io
* highfive.tessel.io
* scbot.tessel.io
* slackin.tessel.io
* start.tessel.io
* t1-start.tessel.io

**Comment by arushi019** [comment]
Hi! 
I am putting in a pull request for this. Is there anything else I should add in the PR other than the information given in comment by @tcr ?

**Comment by Frijol** [comment]
@arushi019 I think @tcr's comment is a good place to start! We can always add more detail in a later PR if needed.

**Comment by tikurahul** [comment]
There is also crash-reporter.tessel.io.

**Comment by arushi019** [comment]
So I can go ahead with this for now?

**Comment by Frijol** [comment]
Yes, please do! We should make sure @tcr and @tikurahul (and probably @reconbot) get a chance to review the PR, as they're more familiar with the webservices we employ

**Comment by arushi019** [comment]
Okay cool!
